### ISSUE TITLE
Improve accessibility for anchor tag 

### ISSUE THREAD

**Issue by amitkate41** [twbs/bootstrap#29650]
**Error Message :**  
Content inserted with CSS is not read by some screen readers, and not available to people  who turn off style sheets.
Not compliant to : 
WCAG 2.0 A F87 Section 508 (2017) A F87

**Content :** 
.custom-file-input:lang(en)~.custom-file-label::after{content:"Browse"}

 a[href]:after {
    content: " (" attr(href) ")";
  }

**Impact on users :**
•NVDA 2018.4 IE11 Windows 10 Reading: Text inserted by CSS content property not read.
•NVDA 2017.3 IE11 Windows 10 Reading: Text inserted by CSS content property not read.
•NVDA 2016.2 IE11 Windows 10 Reading: Text inserted by CSS content property not read.
•JAWS 2018.1811.2 IE11 Windows 10 Reading: Text inserted by CSS content property not read.
•JAWS 18.0.5038 IE11 Windows 10 Reading: Text inserted by CSS content property not read.
•JAWS 17.0.2619 IE11 Windows 10 Reading: Text inserted by CSS content property not read

**Comment by patrickhlauke** [comment]
While it's true that for the custom file uploads, IE11 won't announce the "Browse" ... it also doesn't announce "Browse" for regular `<input type="file">` anyway, so it's not a major change in user experience for the few users still using this now rather outdated browser.

As for the `a[href]:after`, that's part of the print stylesheet, so irrelevant when it comes to screen reader support.
